src/lib/viewers/__tests__/BaseViewer-test.js

```diff
@@ -1858,24 +1863,26 @@ describe('lib/viewers/BaseViewer', () => {
                 destroy: sandbox.stub(),
                 setMode: sandbox.stub(),
             };
-        });
-
-        it('should call toggleAnnotationMode and setMode', () => {
+            base.containerEl = {
```

Comment:
Instead of stubbing the `containerEl` could we create an empty `div` and do assertions on the presence of the class on the `div` itself? That way we can treat it as more black box testing rather than rely on underlying method calls

Reply:
Yep that makes sense